src/markdown-pages/instant-observability/build-a-quickstart/deploy-your-application.mdx

```diff
@@ -0,0 +1,57 @@
+---
+path: '/instant-observability/build-a-quickstart/deploy-your-application'
+title: 'Deploy your application'
+template: 'GuideTemplate'
+description: 'Spin up your demo services to follow along the lab'
+duration: '5 min'
+---
+
+<Callout variant="course" title="Lab">
+
+This procedure is a part of lab that teaches you how to build a quickstart. If you haven't already, check out the [_lab introduction_](/instant-observability/build-a-quickstart)
+
+</Callout>
+
+Before you build a quickstart, you need to spin up demo services. The lab depends on two important services:
+- A python program that mimics the database and also provides functions for CRUD operations. It also makes use of `newrelic_telemetry_sdk` to send telemetry data to New Relic One.
+- A simulator service that generates dummy database traffic so you don't have to manually perform CRUD operations
+
+To spin up demo services, you first need to install [Docker](https://www.docker.com/) and also set **NEW_RELIC_INSERT_KEY** environment variable.
```

Comment:
```suggestion
To spin up these services, you first need to install [Docker](https://www.docker.com/) and also set **NEW_RELIC_INSERT_KEY** environment variable.
```

Reply:
I would add "Install [Docker](https://www.docker.com/)" to the Prerequisites on the intro page.

Reply:
Setting the environment variable is an actual step, so I'd just make a small section here:

## Prepare your environment

On your local machine, set your license key in an environment variable:

```bash
export NEW_RELIC_INSERT_KEY=<YOUR_LICENSE_KEY>
```

Your mock database uses this key to write telemetry data to New Relic. [You can find your New Relic license key in your account settings](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/)